### ISSUE TITLE
radar无法显示刻度，报错

### ISSUE THREAD

**Issue by qingbopro** [apache/incubator-echarts#8301]
### One-line summary [问题简述]
使用type为radar时，无法显示刻度。
axisLabel: {
          show: true   设置为false才行
        },

TypeError: axisModel.getCategories is not a function
    at buildAxisLabel (eval at <anonymous> (reportpreviews.js:1296), <anonymous>:556:35)
    at AxisBuilder.axisTickLabel (eval at <anonymous> (reportpreviews.js:1296), <anonymous>:235:20)
    at AxisBuilder.add (eval at <anonymous> (reportpreviews.js:1296), <anonymous>:137:20)
    at Array.forEach (<anonymous>)
    at Object.each (eval at <anonymous> (reportpreviews.js:594), <anonymous>:298:9)
    at ExtendedClass.eval (eval at <anonymous> (reportpreviews.js:3248), <anonymous>:57:14)
    at Array.forEach (<anonymous>)
    at Object.each (eval at <anonymous> (reportpreviews.js:594), <anonymous>:298:9)
    at ExtendedClass._buildAxes (eval at <anonymous> (reportpreviews.js:3248), <anonymous>:56:12)
    at ExtendedClass.render (eval at <anonymous> (reportpreviews.js:3248), <anonymous>:39:10)

```javascript
option = {
    title: {
        text: '基础雷达图'
    },
    tooltip: {},
    legend: {
        data: ['预算分配（Allocated Budget）', '实际开销（Actual Spending）']
    },
    radar: {
        // shape: 'circle',
        name: {
            textStyle: {
                color: '#fff',
                backgroundColor: '#999',
                borderRadius: 3,
                padding: [3, 5]
           }
        },
        axisLabel: {
          show: true
        },
        indicator: [
           { name: '销售（sales）', max: 6500},
           { name: '管理（Administration）', max: 16000},
           { name: '信息技术（Information Techology）', max: 30000},
           { name: '客服（Customer Support）', max: 38000},
           { name: '研发（Development）', max: 52000},
           { name: '市场（Marketing）', max: 25000}
        ]
    },
    series: [{
        name: '预算 vs 开销（Budget vs spending）',
        type: 'radar',
        // areaStyle: {normal: {}},
        data : [
            {
                value : [4300, 10000, 28000, 35000, 50000, 19000],
                name : '预算分配（Allocated Budget）'
            },
             {
                value : [5000, 14000, 28000, 31000, 42000, 21000],
                name : '实际开销（Actual Spending）'
            }
        ]
    }]
};
```

### Version & Environment [版本及环境]
+ ECharts version [ECharts 版本]:
+ Browser version [浏览器类型和版本]:
+ OS Version [操作系统类型和版本]:

### ECharts option [ECharts配置项]

```javascript
option = {

}

```

**Comment by qingbopro** [comment]
版本 4.1.0

**Comment by qingbopro** [comment]
http://echarts.baidu.com/examples/editor.html?c=radar

**Comment by Clish** [comment]
我今天也遇到同样的问题了, 设置axisLabel: {show: true}就会报错

**Comment by dengzeyuan** [comment]
....我说我怎么配置都不对，我把版本记录完整读了一遍还是没找到升级之后的办法，原来是根本就不支持阿。那文档应该把这个配置删除掉阿

**Comment by 100pah** [comment]
Fixed in #10036
